core/profiles/standard/standard.install

```diff
@@ -339,6 +318,181 @@ function standard_install() {
+      'max_filesize' => '',
+      'max_resolution' => '',
+      'min_resolution' => '',
+      'alt_field' => TRUE,
+      'title_field' => '',
+    ),
+
+    'widget' => array(
+      'type' => 'image_image',
+      'settings' => array(
+        'progress_indicator' => 'throbber',
+        'preview_image_style' => 'thumbnail',
+      ),
+      'weight' => -1,
+    ),
+
+    'display' => array(
+      'default' => array(
+        'label' => 'hidden',
+        'type' => 'image',
+        'settings' => array('image_style' => 'large_cropped', 'image_link' => ''),
+        'weight' => 0,
+      ),
+      'teaser' => array(
+        'label' => 'hidden',
+        'type' => 'image',
+        'settings' => array(
+          'image_style' => 'medium',
+          'image_link' => '',
+          'image_float' => '',
+        ),
+        'weight' => -1,
+      ),
+    ),
+  );
+  field_create_instance($instance);
+
+  // Create a default post to display on front page.
+  $post = new Node(
+    array(
+      'title' => st('Your first post!'),
+      'body' => array(
+        LANGUAGE_NONE => array(
+          array(
+            'value' => st('This is your first post! You may edit or delete it.'),
+            'format' => 'filtered_html'
+          ),
+        ),
+      ),
+      'uid' => 1,
+      'status' => 1,
+      'promote' => 1,
+      'type' => 'post',
+    )
+  );
+  $post->save();
+
+  // Create an "About" page node.
+  $page = new Node(
+    array(
+      'title' => st('About'),
+      'body' => array(
+        LANGUAGE_NONE => array(
+          array(
+            'value' => st('This is a page with static content. You may edit or delete it.'),
+            'format' => 'filtered_html',
+          ),
+        ),
+      ),
+      'uid' => 1,
+      'status' => 1,
+      'promote' => 0,
+      'type' => 'page',
+    )
+  );
+  $page->save();
+
+  // Create an "About" link in the main menu.
+  $item = array(
+    'link_path' => 'node/' . $page->nid,
+    'link_title' => 'About',
+    'weight' => 1,
+  );
+  menu_link_save($item);
+
+  // Creating three default Card nodes.
+
+  // Credits for images.
+  // --- @TODO
+  // --- @TODO
+  // --- @TODO
+
+  $field_image_dir = 'public://field/image';
+  file_prepare_directory($field_image_dir, FILE_CREATE_DIRECTORY);
+
+  $cards = array(
+    'card1-organize.png' => array(
+      'title' => st('Organize your content'),
+      'body' => st('<p>Backdrop CMS includes three pre-defined types of content: Pages, Posts, and Cards. Custom content types can also be added.</p><p><a href="https://docs.backdropcms.org/documentation/content-types">Learn more about content types</a></p>'),
+    ),
+    'card2-card.png' => array(
+      'title' => st('What is a Card?'),
+      'body' => st('<p>A Card is a flexible content item. Cards can be put on any page by placing them into a layout as "Existing content" blocks.</p><p><a href="https://docs.backdropcms.org/documentation/blocks">Learn more about blocks</a></p>'),
```

Comment:
I think this link still needs to be updated? Currently pointing to blocks page.

Reply:
This description doesn't seem like it answers the question "what is a card?" Attempting a reword...

Thought process: 
* I don't think we need to say a Card is content, since we said that on the `Organize` card. 
* Is specifically mentioning the "Existing content" block helpful here? Or is that too much detail?
* If we do mention the  block placement, we shouldn't exclude a view as another option. Stealing text from the node type description.. 

`A Card is intended to be viewed only as part of another page. Place them individually as Blocks, or grouped as in a View.`

Reply:
Ugh, `is intended to be viewed only as` still sounds cumbersome. trying again...

`A Card can only be seen as part of another page. Place them individually as Blocks, or grouped as in a View.` This leaves room for the type of block... 

`A Card can only be seen as part of another page. Place them individually as "Existing content" Blocks, or grouped as in a View.`